### ISSUE TITLE
Compile the workspace with xcodebuild

### ISSUE THREAD

**Issue by jayztemplier** [CocoaPods/CocoaPods#270]
Hi,
I would like to know if there is a way to compile to workspace in command line ? 

Actually, this command line works : 
xcodebuild -workspace 'Project.xcworkspace' -scheme 'Project'

The problem is that it saves the build into the DeriveData folder with a weird name.
So I tried this command line :
 xcodebuild -workspace 'New Relic.xcworkspace' -scheme 'New Relic' CONFIGURATION_BUILD_DIR='build'

The problem with this one is that I always have this error :
ld: library not found for -lPods
clang: error: linker command failed with exit code 1 (use -v to see invocation)
*\* BUILD FAILED **

Do you have an idea ? 

Thanks !
Jeremy.

**Comment by brajbong** [comment]
I am using Jenkins and the Xcode plugin. As suggested by @smistry-toushay, I set the SYMROOT and OBJROOT variables. I now do not get any error related to "ld: library not found for -lPods". However, Jenkins bailed out with the following message-
Check dependencies
error: Unable to create directory: /build (Permission denied)
error: Unable to create directory: /build (Permission denied)

Apparently, when I set SYMROOT to $(PWD)/build, it becomes /build (do not know why) and the build process wants to create the following folder structure, for which obviously I have no permission.

/bin/mkdir -p /build/<target_name>.build/Release-iphoneos/<target_name>.build

Any idea as to what I might be doing wrong here? Why does SYMROOT point to /build. I believe it should have pointed to the build folder in the project workspace? What's the significance of using $PWD/build in SYMROOT?
